### DISCUSSION TITLE
Running Supervised PCA

### DISCUSSION THREAD

**Discussion by adamgayoso** [satijalab/seurat#4987] [Q&A] [answered]
Hello,

I'm interested in building my own CITE-seq reference using WNN and supervised PCA. However, I can't seem to find the function reference for RunSPCA in the [docs](https://satijalab.org/seurat/reference/index.html), nor do I see how to compute it in the WNN or query mapping tutorials.

Any help is appreciated.

Update: I found it in the multimodal reference mapping tutorial, but it's pretty deep in there. I still don't see it in the docs I linked above, but can search it from google and get something that looks like seurat docs?

Thanks,
Adam

**Comment by saketkc** [comment, marked as answer]
Hi Adam, the documentation is [here](https://satijalab.org/seurat/reference/runspca). It's missing from the reference because of a context missing, but should be in develop soon.

**Reply by timoast** [reply]
Just to add, you can find documentation for any function by typing `?<function_name>` in R, eg `?RunSPCA`

**Reply by adamgayoso** [reply]
Thanks, though I will say it would be nice to better highlight in the tutorials this functionality.
